en/advanced_features/traffic_avoidance_adsb.md

```diff
@@ -75,20 +75,19 @@ Configure the action when there is a potential collision using the parameter bel
 
 Parameter | Description
 --- | ---
-<span id="NAV_TRAFF_AVOID"></span>[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Enable traffic avoidance mode specify avoidance response. 0: Disable, 1: Warn only, 2: Return mode, 3: Land mode.
-<span id="NAV_TRAFF_A_RADM"></span>[NAV_TRAFF_A_RADM](../advanced_config/parameter_reference.md#NAV_TRAFF_A_RADM) | Set traffic avoidance distance for *manned* aviation
-<span id="NAV_TRAFF_A_RADU"></span>[NAV_TRAFF_A_RADU](../advanced_config/parameter_reference.md#NAV_TRAFF_A_RADU) | Set traffic avoidance distance for *unmanned* aviation
-
+<a id="NAV_TRAFF_AVOID"></a>[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID)  | Enable traffic avoidance mode specify avoidance response. 0: Disable, 1: Warn only, 2: Return mode, 3: Land mode.
+<a id="NAV_TRAFF_A_HOR"></a>[NAV_TRAFF_A_HOR](../advanced_config/parameter_reference.md#NAV_TRAFF_A_HOR)  | Set NAV TRAFFIC AVOID horizontal distance
+<a id="NAV_TRAFF_A_VER"></a>[NAV_TRAFF_A_VER](../advanced_config/parameter_reference.md#NAV_TRAFF_A_VER)  | Set NAV TRAFFIC AVOID vertical distance
+<a id="NAV_TRAFF_COLL_T"></a>[NAV_TRAFF_COLL_T](../advanced_config/parameter_reference.md#NAV_TRAFF_COLL_T) | Estimated time until collision based on relative speed of traffic and UAV
 
 ## Implementation
 
 PX4 listens for valid transponder reports during missions.
 
-If a valid transponder report is received, PX4 first uses the transponder position and heading information to estimate whether the vehicles will share a similar altitude before they pass each other.
-If they may then PX4 it estimates how the closest distance between the path to the next waypoint and the other vehicles predicted path.
-If the crossing point is less than the configured distance for altitude and path, the [Traffic Avoidance Failsafe](../config/safety.md#traffic-avoidance-failsafe) action is started, and the vehicle will either warn, land, or return.
-The detection distance can be configured separately for manned and unmanned aviation.
-
+If a valid transponder report is received, PX4 first uses the traffic transponder information to estimate whether the traffic heading and height indicates there will be an intersection with the airspace of the UAV.
+The UAV airspace consists of a surrounding cylinder defined by [NAV_TRAFF_A_HOR](#NAV_TRAFF_A_HOR) and [NAV_TRAFF_A_VER](#NAV_TRAFF_A_VER) with the UAV at it's center.
```

Comment:
@akkawimo @junwoo091400 Is the suggestion below correct? -i.e. is this a radius as I expect? I am going to assume so based on the diagram in the PR, but if this is not true, please let me know.

```suggestion
The UAV airspace consists of a surrounding cylinder defined by the radius [NAV_TRAFF_A_HOR](#NAV_TRAFF_A_HOR) and height [NAV_TRAFF_A_VER](#NAV_TRAFF_A_VER), with the UAV at it's center.
```